sim_obstacles_detector:
  obstacle_detect_loop_freq: 10.0
  robot_frame: 'robot_base_link'
  detector_range: 2.0
  cov_meas_pos:
    x: 0.0001
    y: 0.0001
    z: 0.00001
  cov_meas_siz:
    R: 0.0001
    h: 0.000001
  
